js/languages/en-US.json

```diff
@@ -87,6 +87,57 @@
     "userLoading": "Loading...",
     "userNotFound": "Information for %{guid} was not found on the network."
   },
+  "about": {
+    "_linkText": "About OpenBazaar",
```

Comment:
I'm still wondering :) ... Is it critical to maitain this '_ ' preface to some translation keys? Even after your explanation, I still don't understand the reasoning. My concern is that other developers / translators (like me) see this convention and have no idea what is going on.